File: week1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)

{
    //prompt user for input

    int y;
    do
    {
        y = get_int("height:");
    }
    while (y < 1 || y > 8);

    //build blocks
        int r = y-1;

        //build (vertical)
        for (int i = 0 ; i < y; i++)
        {
            //build horizontal
                //number of dots
                for (int j = r; j > 0; j--)
                {
                 printf(" ");

                }
                //number of hashes
                for (int k = r; k < y; k++)
                {
                    printf("#");
                }
                printf("\n");
                r--;
        }
}

File: week6python/pset6/marioless/mario.py
import cs50

height = cs50.get_int("Height: ")
while height < 1 or height > 8:
    height = cs50.get_int("Height: ")
for i in range(height, 0, -1):
    print(" " * (i - 1), end="")
    print("#" * (height + 1 - i))